src/app/contentaccess/metering/metering.html

```diff
@@ -90,7 +102,7 @@
                   <a
                     href="#contentaccess/subscription?partnerId={{partnerId}}&redirect={{redirect}}"
                     class="btn btn-success btn-success-tair"
-                    >SUBSCRIBE</a
+                    >SUBSCRIBE OR BUY</a
```

Comment:
The display of this one is a bit weird. The current button width is insufficient for the text